### ISSUE TITLE
Clicking checkbox/radio labels should fire onClick messages

### ISSUE THREAD

**Issue by jhrcek** [rundis/elm-bootstrap#88]
@rundis First of all thank you very much for migrating to latest Bootstrap 4.0.0, I Imagine it was not a small undertaking.

Today I migrated my app to this new version of this library and everything seems to work nice. I just noticed one undesirable change from version 3.0.0:
in 3.0.0 when I click on a label associated with a radio / checkbox, it fires its on click event as if I just clicked the button / checkbox itself.
in 4.0.0 the onClick event is only fired when I click the checkbox / radio button itself (but does nothing when clicking the label).

Could you please bring that 3.0.0 behavior back? I'd say it's consider [good practice](https://schoberg.net/2008/10/making-better-forms-clickable-text-for-radio-button-and-checkbox-fields/) and it's also the default behavior when you use ["label for"](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio) in vanilla HTML..

**Comment by rundis** [comment]
Twbs changed how checkbox an radios needs to renderded (label and attribute as siblings, rather than label wrapping input). Short story, you need to pass in an id. If you just use Radio.id or Checkbox.id options. Itwill set the inputs id and the labels for attribute.  Have you tried setting the id ?

**Comment by rundis** [comment]
As a side note: I'm had a nagging feeling this would come up, and it didn't take long!
I did consider adding id as an explicit function param. However

`checkbox : List (Option msg) -> String -> String -> Html msg` didn't look to appealing and having an Id type would also make the api more cumbersome. So I settled for id as an Option and tried to make it move obvious to use it through docs. In hindsight I should probably have tried to figure out an alternative API.
